src/features/rewards/hero/index.tsx

```diff
@@ -7,15 +7,19 @@ import { StyledHero } from './style'
 
 export interface Props {
   id?: string
+  roundBorder?: boolean
```

Comment:
I think that we should be using isMobile or something like this flag, so that we can use it for other changes as well if needed

Reply:
Sure, can change